admin/src/pages/UserSearch.vue

```diff
@@ -70,11 +82,11 @@ export default {
   },
   methods: {
     unconfirmedRegisterMails() {
-      this.filterCheckedEmails = this.filterCheckedEmails ? null : true
+      this.filterByActivated = this.filterByActivated === null ? false : null
```

Comment:
Not sure why you set filterByActivated on null and not on true?
Like this:
```suggestion
      this.filterByActivated = this.filterByActivated ? false : true
```

Reply:
This would get wrong, because we switsch between `null` and `false`.

I like that we can see explizit that we switsch between `null` and `false`.
And then the logic works already if we include `true` in the future as well.